apps/common-app/src/examples/RuntimeTests/tests/animations/withSequence/arrays.test.tsx

```diff
@@ -99,23 +112,20 @@ describe('withSequence animation of array', () => {
           animationNumber={animationNumber}
         />,
       );
-      const componentOne = getTestComponent(COMPONENT_REF.first);
-      const componentTwo = getTestComponent(COMPONENT_REF.second);
-      const componentThree = getTestComponent(COMPONENT_REF.third);
-      const margin = 30;
 
-      await wait(200 + DELAY / 2);
-      expect(await componentOne.getAnimatedStyle('left')).toBe(finalValues[0] + margin, ComparisonMode.PIXEL);
-      expect(await componentTwo.getAnimatedStyle('left')).toBe(finalValues[1] + margin, ComparisonMode.PIXEL);
-      expect(await componentThree.getAnimatedStyle('left')).toBe(finalValues[2] + margin, ComparisonMode.PIXEL);
-      await wait(300 + DELAY / 2);
-      expect(await componentOne.getAnimatedStyle('left')).toBe(middleValues[0] + margin, ComparisonMode.PIXEL);
-      expect(await componentTwo.getAnimatedStyle('left')).toBe(middleValues[1] + margin, ComparisonMode.PIXEL);
-      expect(await componentThree.getAnimatedStyle('left')).toBe(middleValues[2] + margin, ComparisonMode.PIXEL);
-      await wait(200 + DELAY);
-      expect(await componentOne.getAnimatedStyle('left')).toBe(finalValues[0] + 20 + margin, ComparisonMode.PIXEL);
-      expect(await componentTwo.getAnimatedStyle('left')).toBe(finalValues[1] + 20 + margin, ComparisonMode.PIXEL);
-      expect(await componentThree.getAnimatedStyle('left')).toBe(finalValues[2] + 20 + margin, ComparisonMode.PIXEL);
+      const snapshotNamePrefix = `from_${startValues.join('_')}_through_${middleValues.join('_')}_to_${finalValues.join(
+        '_',
+      )}_animation_${animationNumber}`.replace('-', 'min');
+
+      await wait(1000); // waitForAnimationUpdates doesn't work with multiple view recordings
```

Comment:
so we can either compute the sum of frames or split the tests into a more granular level

Reply:
>split the tests into a more granular level

I've incorporated this solution in most of the cases, yet here I would like to avoid it.
This tests verifies simultaneous animation of array of elements. We could change this and animate 3 various numerical props of a single component, yet that would be a completely different case.

>compute the sum of frames

I did consider such a solution, but unfortunately it turned out to be much more complicated than I thought and decided to abandon this approach.

I am afraid that keeping the `wait` command is the best solution, although it is still not perfect.